docs: opening invoice creation tool (#181)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/accounts/opening-invoice-creation-tool.md

<!-- add-breadcrumbs -->
# Opening Invoice Creation Tool

> The Opening Invoice Creation Tool allows importing data of outstanding purchase or sales invoices into ERPNext. This specific tool is used in place of Data Import Tool for cases where item data is irrelevant and outstanding balances against customer/supplier is to be imported into ERPNext.

Reviewer: ```suggestion
The Opening Invoice Creation Tool allows importing data of outstanding Purchase or Sales Invoices into ERPNext. This specific tool is used in place of the Data Import Tool for cases where the Item data is irrelevant and outstanding balances against Customers/Suppliers are to be imported into ERPNext.
```

> To access the tool, go to:

Reviewer: ```suggestion
To access the Opening Invoice Creation Tool, go to:
```

> Home > Accounting > Opening and Closing > Opening Invoice Creation Tool

## 1: How to import Opening Invoices

1. Select the Company to which you want to import opening balances.

2. Select the Invoice Type. Selecting Sales or Purchase will generate Sales Invoices or Purchase Invoices respectively.

> 3. Checking Create Missing Party will automatically create customers or suppliers if missing according to the name provided in Party column.

Reviewer: ```suggestion
3. Checking the "Create Missing Party" checkbox will automatically create customers or suppliers if missing according to the name provided in the Party column.
```

    <img class="screenshot" alt="Opening Invoice Creation Tool" src="{{docs_base_url}}/assets/img/setup/opening-invoice-creation-tool.png">

> 4. Fill up the Invoices table. It consists of following fields:

Reviewer: ```suggestion
4. Fill up the Invoices table. It consists of the following fields:
```

>     - **Party**: You can select a customer/supplier or enter name of a new one which will be automatically created.

Reviewer: ```suggestion
    - **Party**: You can select an existing Customer/Supplier or enter the name of a new one which will be automatically created.
```

    - **Posting Date**: The date at which the invoice will be posted.
    - **Due Date**: The date after which the invoice will be overdue.
    - **Item Name**: (Optional) The item name entered here will be shown in the invoice item table.
    - **Outstanding Amount**: The outstanding amount of the invoice.

> > **Tip**: You can click the download button to download an excel sheet which you can fill up easily with appropriate data. If you have downloaded the excel sheet, then use the upload button to upload it. Once you upload the sheet, the table will be filled with appropriate data rows.

Reviewer: ```suggestion
> **Tip**: You can click the download button to download an excel sheet that you can fill up easily with appropriate data. If you have downloaded the excel sheet, then use the Upload button to upload it. Once you upload the sheet, the table will be filled with appropriate data rows.
```

> > **Tip**: You can click the download button to download an excel sheet which you can fill up easily with appropriate data. If you have downloaded the excel sheet, then use the upload button to upload it. Once you upload the sheet, the table will be filled with appropriate data rows.

Author: ```suggestion
> **Tip**: You can click the download button to download an excel sheet that you can fill up easily with appropriate data. If you have downloaded the excel sheet, then use the Upload button to upload it. Once you upload the sheet, the table will be filled with appropriate data rows.
```
